app/models/account.rb

```diff
@@ -54,7 +57,7 @@ class Account < ApplicationRecord
 
   # Local user validations
   with_options if: 'local?' do
-    validates :username, format: { with: /\A[a-z0-9_]+\z/i }, uniqueness: { scope: :domain, case_sensitive: false }, length: { maximum: 30 }
+    validates :username, format: { with: /\A[a-z0-9_]+\z/i }, uniqueness: { scope: :domain, case_sensitive: false }, length: { maximum: 30 }, exclusion: {in: VALID_RESERVED_WORDS}
```

Comment:
This will blow up for users that already have accounts with these names, like support@mastodon.social, right? Ideally this would *only* be a check that affects the creation of users.

Also, I don't think this prevents someone from creating "Admin" or "HELP".